source/features/highest-rated-comment.tsx

```diff
@@ -63,6 +95,22 @@ function init(): false | void {
 	}
 }
 
+function getWatchedReactions(): HTMLElement[] {
+	return positiveReactions.flatMap(reaction => select.all(`${COMMENT_SELECTOR} ${reaction}`));
+}
+
+function getNegativeReactions(reactionBox: HTMLElement): HTMLElement[] {
+	return negativeReactions.flatMap(reaction => select.all(`${reaction}`, reactionBox));
+}
+
+function getPositiveReactions(reactionBox: HTMLElement): HTMLElement[] {
+	return positiveReactions.flatMap(reaction => select.all(`${reaction}`, reactionBox));
+}
```

Comment:
You can avoid these helper functions and just query the selector once:

```js
select.all(positiveReactions.join(','), reactionBox)
```

Reply:
ok

Reply:
I prefer to leave the helper functions in though for future refactoring and readability.